docker/Dockerfile.sys-mgmt-agent

```diff
@@ -7,24 +7,42 @@
 #
 
 FROM golang:1.11.2-alpine3.7 AS builder
-WORKDIR /go/src/github.com/edgexfoundry/edgex-go
+# WORKDIR /go/src/github.com/edgexfoundry/edgex-go
 
 # The main mirrors are giving us timeout issues on builds periodically.
 # So we can try these.
 RUN echo http://nl.alpinelinux.org/alpine/v3.7/main > /etc/apk/repositories; \
     echo http://nl.alpinelinux.org/alpine/v3.7/community >> /etc/apk/repositories
 
+WORKDIR /go/src/github.com/docker-compose-executor
+# RUN apk update && apk add git
+
 RUN apk update && apk add make glide git
 COPY . .
 
-RUN make prepare
-RUN make cmd/sys-mgmt-agent/sys-mgmt-agent
+# RUN make prepare
+# RUN make cmd/sys-mgmt-agent/sys-mgmt-agent
+
+# Build the golang "executor" bits (an SMA-related reference implementation) by fetching the source from Github.
+RUN git clone https://github.com/akramtexas/docker-compose-executor
```

Comment:
Clone this to /go/src/github.com/docker-compose-executor since that what you have WORKDIR set as.  You may need to reset WORKDIR for building each of docker-compose-executor and SMA.